docs/content/guides/getting-started/grid-size.md

```diff
@@ -46,6 +46,11 @@ If you'd like to observe it, you can define the dimensions in the configuration
     height: 100 // For a better compatibility we convert number into pixels
 }
 ```
+::: only-for react
+```jsx
+  <HotTable height={100} width={100} /> 
```

Comment:
its shown in snippet above (https://github.com/handsontable/handsontable/pull/9787/files#r953409190)

Reply:
If the snippet above stays - true; if it doesn't, this will need to be extended. We'll see how https://github.com/handsontable/handsontable/pull/9787#discussion_r951224309 turns out.

Reply:
will edit both javascript and react versions